src/util.py

```diff
@@ -291,25 +323,33 @@ def get_relevant_ac_versions(fenix_repo, ac_repo):
 def validate_as_version(v):
     """Validate that v is in the format of 63.0.2. Returns v or raises an exception."""
     if not re.match(r"^\d+\.\d+\.\d+$", v):
-        raise Exception(f"Invalid A-S version format {v}")
+        raise Exception(f"Invalid version format {v}")
     return v
 
 
-def match_as_version(src):
```

Comment:
I made these project specific because all projects seem to have different version formats and that made it more explicit what was checked. Maybe keep the `_as_` in there?